pkg/deployment/deployment_inspector.go

```diff
@@ -45,10 +46,33 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	twentySeconds = time.Second * 20
```

Comment:
Call it in more related to use-case way, like PerNodeReconcilationTimeout

Reply:
changed